src/avlizer_confluent.erl

```diff
@@ -303,10 +303,18 @@ terminate(_Reason, _State) ->
 
 %%%_* Internals ================================================================
 
-get_registry_url() ->
-  {ok, #{ schema_registry_url := URL
-        }} = application:get_env(?APPLICATION, ?MODULE),
-  URL.
+get_registry_base_request() ->
+  case application:get_env(?APPLICATION, ?MODULE) of
```

Comment:
if we do it like this:
```
{ok, #{ schema_registry_url := URL
       }} = Cfg = application:get_env(?APPLICATION, ?MODULE),
Auth = maps:get(Cfg, schema_registry_auth, false)
{URL, get_registry_headers(Auth)}
```
then there is no need to have a `get_registry_headers/0`